index.bs

```diff
@@ -1,9 +1,19 @@
-<h1>Web Authentication:<br>An API for accessing Public Key Credentials<br>[Level 1]</h1>
+<!--   ***********  Web Authentication - Level 1 - Spec source file ***********
```

Comment:
This change doesn't seem related to algorithm changes. Make it a separate PR.

Reply:
well, we have done such while-i'm-in-there editors' notes since we began this project and if we waited to do separate PRs for ever little thing they'd never get done.  if it's innocuous and not obviously incorrect, then we've been incorporating them.